static/js/components/GroupSources.jsx

```diff
@@ -196,28 +188,57 @@ const GroupSources = ({ route }) => {
     );
   };
 
+  const renderAlias = (dataIndex) => {
+    const { id: objid, alias } = sources[dataIndex];
+
+    return (
+      <a href={`/source/${objid}`} key={`${objid}_alias`}>
+        {alias}
+      </a>
+    );
+  };
+
   // This is just passed to MUI datatables options -- not meant to be instantiated directly.
 
   const renderRA = (dataIndex) => {
     const source = sources[dataIndex];
-    return (
-      <div key={`${source.id}_ra`}>
-        {source.ra}
-        <br />
-        {ra_to_hours(source.ra)}
-      </div>
-    );
+    return <div key={`${source.id}_ra`}>{source.ra}</div>;
```

Comment:
Might be a good idea to specify .toFixed() values for these numerical columns (RA, Dec, redshift) - we don't have to round too aggressively (could be like .toFixed(6) maybe?) but doesn't hurt to just have that in there in case the values are very long I think.